### PR TITLE
Turn off transitive in copy dependencies

### PR DESCRIPTION
**Pull request by AlexanderPrendota**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -43,15 +45,18 @@ allprojects {
...
+    kotlinDependency("com.fasterxml.jackson.core:jackson-annotations:2.10.0")
+    // Kotlin libraries
     kotlinDependency("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion")
+    kotlinDependency("org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion")
     kotlinDependency("org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion")
-    kotlinDependency("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.5") {
-        exclude(group = "org.jetbrains.kotlin", module = "kotlin-stdlib")
-    }
+    kotlinDependency("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
+    kotlinDependency("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
```
Comment (dsavvinov) [inline build.gradle.kts:58]: Am I missing something, or stdlilb is added twice?

File: build.gradle.kts [hunk 2]
```diff
@@ -8,7 +8,9 @@ group = "com.compiler.server"
 version = "$kotlinVersion-SNAPSHOT"
 java.sourceCompatibility = JavaVersion.VERSION_1_8
 
-val kotlinDependency: Configuration by configurations.creating
+val kotlinDependency: Configuration by configurations.creating {
+    isTransitive = false
+}
 val kotlinJsDependency: Configuration by configurations.creating
```
Comment (dsavvinov) [inline build.gradle.kts:14]: Is it intentional that we don't disable transitive dependencies for JS?

Comment (AlexanderPrendota) [reply build.gradle.kts:14]: I have no idea about JS. @dsavvinov I guess kotlin-plugin does not affect JS dependencies. Shall I add?

File: build.gradle.kts (hunk 1 above)
Comment (AlexanderPrendota) [reply build.gradle.kts:58]: Oh, You are right. Will fix =)

File: build.gradle.kts (hunk 2 above)
Comment (dsavvinov) [reply build.gradle.kts:14]: I think we'd better disable them for JS as well, just for the sake of consistency (as far as I can see, it doesn't change anything currently; at most you'd need to add dependency to `kotlin-stdlib-common`)

Comment (AlexanderPrendota) [reply build.gradle.kts:14]: Will add. thanks)

**Pull request merged**